src/plugins/intel_cpu/src/emitters/snippets/x64/jit_fill_emitter.cpp

```diff
@@ -59,7 +59,11 @@ void jit_fill_emitter::emit_isa(const std::vector<size_t> &in, const std::vector
     Vmm src_vmm = Vmm(in[0]);
     Vmm dst_vmm = Vmm(out[0]);
 
-    if (is_full_reg())
+    const size_t supported_et_size = 4;
+    const auto register_capacity = (src_vmm.getBit() / 8) / supported_et_size;
+    if (offset == register_capacity)
```

Comment:
BTW you can optimize more 😃 If  `src_vmm.getIdx() == dst_vmm.getIdx()`, we may do nothing (instead of useless moving from the reg to the same reg)